ktor-server/ktor-server-plugins/ktor-server-i18n/jvm/test/io/ktor/tests/i18n/TranslationTest.kt

```diff
@@ -67,6 +68,29 @@ class TranslationTest {
         assertEquals("English Key", body)
     }
 
+    @Test
```

Comment:
I think it's worth adding a test with `StatusPages` plugin

Reply:
Added